### ISSUE TITLE
Why our reproduced accuracy is so much higher than in the paper.

### ISSUE THREAD

**Issue by huzj01** [MILVLG/mcan-vqa#45]
Why our reproduced accuracy is so much higher than in the paper.

Overall Accuracy is: 81.02

Per Answer Type Accuracy is the following:
other : 73.65
yes/no : 95.62
number : 66.90

**Comment by MIL-VLG** [comment]
Maybe you include the test(val) set into the training set?
